IFT-Explainer.md

```diff
@@ -0,0 +1,161 @@
+and **network cost** (impact of latency on time to render)
+were considered.
+
+These initial approaches informed the current specification,
+but the WG ended up discarding one and substantially rewriting the other.
+
+### Range Request
+
+The Range Request method relied on the existing HTTP Range Request functionality
+and therefore could be used with **any HTTP server**.
+For best efficiency, the font needed to be 
+[re-ordered](https://w3c.github.io/IFT/RangeRequest.html#font-organization)
+before upload to the server.
+
+The client still needed to be updated to support this method.
+
+Work on Range Request was discontinued because:
+
+- Performance was insufficient in general
+- Only glyph outlines were subsetted
+- On slower networks, Range Request was [worse then no subsetting at all](https://www.w3.org/TR/PFE-evaluation/#conclusions-shaping)
+
+### Patch Subset
+
+The Patch Subset method required the server to 
+[**dynamically respond to a PatchRequest**](https://www.w3.org/TR/2023/WD-IFT-20230530/#handling-patch-request)
+by validating the request,
+computing a binary patch between the current, subsetted font on the client
+and the desired subset of the original font,
+and then sending the patch,
+which the client applied to produce a new, enlarged subset font.
+
+It therefore required new server capabilities,
+in addition to client changes.
+
+Work on Range Request was discontinued because, despite
+[median byte reductions of 90% for large fonts](https://www.w3.org/TR/PFE-evaluation/#analysis-cjk),
+
+- The dynamic subsetting severely impacted CDN **cache performance**
+- There was no subsetting of _design axes_, an unacceptable result with **variable fonts** becoming ubiquitous
+- Subsetting fonts with **complex interactions between glyphs** was challenging
+- Non-outline data was not subsetted, and can be significant (e.g. Color fonts)
+- Requiring an intelligent, dynamic server hindered widespread deployment
+- Very fine-grained subsetting [might be a privacy violation](https://www.w3.org/TR/2023/WD-IFT-20230530/#content-inference-from-character-set)
+
+## Incremental Font Transfer
+
+The [current specification](https://w3c.github.io/IFT/Overview.html) 
+draws on the Patch Subset concept
+of patching a font to provide more data.
+However, instead of requiring custom patch generation for each user,
+the initial font has two new [_Patch Map_ tables](https://w3c.github.io/IFT/Overview.html#patch-map-dfn)
+which map each subset definition to **the link** of the relevant patch.
+Links are stored as RFC 6570 [URI Templates](https://www.rfc-editor.org/rfc/rfc6570).
+
+Thus, static hosting of files (base fonts, and patches) is easy,
+and cache performance is good
+because patches are shared between users.
+
+Both **independent** (commutative) and **dependent** (non-commutative)
+patches are [supported](https://w3c.github.io/IFT/Overview.html#font-format-extensions).
+
+In addition, patches can add or extend design axes

```

Comment:
This is true but because of the two patch types maybe a bit less true of this system than the last one (in that to **add** an axis you'll typically wind up downloading some duplicate glyph data). 

@garretrieger is this OK or should we soft-petal it more?

Reply:
Fixed